jujugui/static/gui/src/app/components/deployment-flow/credential/add/test-add.js

```diff
@@ -50,7 +50,9 @@ describe('DeploymentCredentialAdd', function() {
             title: 'Client e-mail address'
           }, {
             id: 'private-key',
-            title: 'Client secret'
+            title: 'Client secret',
```

Comment:
This was actually supposed to have been changed, must have missed it, oops, fixed!